mec/zp.py

```diff
@@ -536,7 +536,7 @@ def _do_load(self, suffix):
             log.debug('Load took %.1f seconds', duration)
             data = json.loads(raw_data)
             if 'status' in data:
-                status = data['status']
+                status = int(data['status'])
```

Comment:
I can confirm this is a change in the json response.  As data['status'] is returned and used elsewhere it's probably best to set it back to an int here, I'll push a commit.